docs/CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,179 @@
+
+1. The contributor is nominated by opening a PR against the appropriate repository by the existing reviewers, committers and maintainers, which adds their GitHub username to the OWNERS file for one or more directories.
+2. At least 1/3 members of the reviewers, committers, and maintainers who are already Approvers, approve the PR.
+
+### Committer
+Description: A Committer has responsibility for specific code, documentation, test, or other project areas. They are collectively responsible, with other Committers, for reviewing all changes to those areas and indicating whether those changes are ready to merge. They have a track record of contribution and review in the project.
+
+Committers are responsible for a "specific area." This can be a specific code directory, driver, chapter of the docs, test job, event, or other clearly-defined project component that is smaller than an entire repository or subproject. Most often it is one or a set of directories in one or more Git repositories. The "specific area" below refers to this area of responsibility.
+
+Committers have all the rights and responsibilities of a reviewer, plus:
+
+* Responsibilities include:
+    * Reviewing most Pull Requests against their specific areas of responsibility
+    * 5 accepted PRs with code changes (not just docs or typo fixes) per year
+    * Helping other contributors become Committers
+* Requirements:
+    * Experience as a Reviewer for at least 3 months
+    * Has reviewed, or helped review, at least 15 Pull Requests
+    * Has analyzed and resolved test failures in their specific area
+    * Has demonstrated an in-depth knowledge of the specific area
+    * Commits to being responsible for that specific area
+    * Is supportive of new and occasional contributors and helps get useful PRs in shape to commit
+* Additional privileges:
+    * Has GitHub or CI/CD rights to approve pull requests in specific directories
+    * Can recommend and review other contributors to become Committers
+    * Is listed as Committers in the [OWNERS file](./OWNER.md) under the WasmEdge/WasmEdge repos.
+  
+
+The process of becoming a Committer is:
+
+1. The contributor is nominated by opening a PR against the appropriate repository by the existing reviewers, committers and maintainers, which adds their GitHub username to the OWNERS file for one or more directories.
+2. At least 1/2 members of the reviewers, committers, and maintainers that owns that repository or main directory, who are already Approvers, approve the PR.
+
+
+### Maintainer
+<!-- In the simplest and most common project structures, projects have a single pool of "maintainers" who are collectively responsible for the entire project.  This example defines that role for your project.  If your project has a more complex structure, see the list of specific maintainer roles you might want to define, below. -->
+<!--TODO: project leads to fill in exact details of this role for your project-->
```

Comment:
What's the intention behind keeping these comments here?

Reply:
I can delete it.